src/sugar.jl

```diff
@@ -430,11 +430,24 @@ macro accessor(ex)
     end |> esc
 end
 
+### shortcuts:
+# optic macro
+const var"@o" = var"@optic"
+
+# Elements, Properties
+const ∗ = Elements()
+const ∗ₚ = Properties()
+IndexLens(::Tuple{Elements}) = Elements()
+IndexLens(::Tuple{Properties}) = Properties()
```

Comment:
This looks much better than doing it at "parse time". Can you add a test where the user defines these symbols?

Reply:
Optionally we could add a layer of indirection and instead of overloading the `IndexLens` constructor with something that is not an IndexLens, do this dispatch in another function `index_like_lens`.